doc/release-notes.md

```diff
@@ -19,44 +19,236 @@ dashd/dash-qt (on Linux).
 
 ## Downgrade warning
 
-### Downgrade to a version < v22.0.0
+### Downgrade to a version < v23.0.0
+
+Downgrading to a version older than v23.0.0 is not supported, and will
+require a reindex.
+
+# Notable changes
+
+## EvoDB migration
+
+This release introduces a new internal format for masternode state data to support extended addresses and make future updates of this data seamless. Nodes will automatically migrate to the new format. Downgrading to an earlier version would require a full reindex. #6813
+
+## Breaking Change: Block Filter Index Format Update
+
+The compact block filter index format has been updated to include Dash special transaction data, providing feature parity with bloom filters for SPV client support. This change is incompatible with existing blockfilter indexes. Existing blockfilter indexes will automatically be re-created with the new version.
+
+- The blockfilter index now includes fields from Dash special transactions:
+  - ProRegTx (masternode registration)
+  - ProUpServTx (masternode service updates)
+  - ProUpRegTx (masternode operator updates)
+  - ProUpRevTx (masternode revocation)
+  - AssetLockTx (platform credit outputs)
+- A versioning system has been added to detect incompatible indexes on startup
+- The index version is now 2 (previously unversioned)
+
+### Benefits
+
+- SPV clients can now detect and track Dash-specific transactions
+- Feature parity between bloom filters and compact block filters
+- Protection against serving incorrect filter data to light clients
+
+## Other notable changes
+
+* Performance of block validation in Dash Core has been significantly improvement, index from
+  scratch and reindex are up to 20% faster compare to v22.x.
+* To help prevent fingerprinting transactions created by the Dash Core wallet, change output
+  amounts are now randomized. (#6685)
+* Dash Core will no longer migrate EvoDb databases generated in v19 and v20, users upgrading
+  from these versions are recommended to run `-reindex` to rebuild all databases and indexes. (#6579)
+
+## Updated REST APIs
+
+- The `/headers/` and `/blockfilterheaders/` endpoints have been updated to use
+  a query parameter instead of a path parameter to specify the result count. The
+  count parameter is now optional and defaults to 5 for both endpoints. The old
+  endpoints are still functional, and have no documented behaviour change. (#24098)
```

Comment:
nit: here and in a couple more places - should maybe reference the PR that backported this, not the original one